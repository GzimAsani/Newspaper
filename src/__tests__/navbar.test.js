import { BrowserRouter} from 'react-router-dom';
import { create } from 'react-test-renderer';
import Navbar from '../components/navbar';

test('renders correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});