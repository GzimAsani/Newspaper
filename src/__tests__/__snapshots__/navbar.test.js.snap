// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=""
>
  <nav
    className="navbar navbar-expand-lg navbar-light bg-light"
  >
    <div
      className="container-fluid"
    >
      <button
        className="navbar-brand text-success btn btn-outline-info "
        type="button"
      >
        <a
          href="/posts"
          onClick={[Function]}
        >
           BACK 
        </a>
      </button>
      <button
        type="button"
      >
        <a
          href="/posts"
          onClick={[Function]}
        >
          POSTS
        </a>
      </button>
    </div>
    <div />
  </nav>
</div>
`;
